src/game_api/custom_types.cpp

```diff
@@ -551,7 +551,8 @@ std::span<const ENT_TYPE> get_custom_entity_types(CUSTOM_TYPE type)
             "ENT_TYPE_ITEM_DMCRATE",
             "ENT_TYPE_ITEM_PRESENT",
             "ENT_TYPE_ITEM_GHIST_PRESENT",
-            "ENT_TYPE_ITEM_ALIVE_EMBEDDED_ON_ICE");
+            "ENT_TYPE_ITEM_ALIVE_EMBEDDED_ON_ICE",
+            "ENT_TYPE_ITEM_POT");
```

Comment:
This is technically is not correct, can you maybe create a new type for this, could then also include coffin

Reply:
Neither is present as far as I know. I still think "can contain arbitrary item" would be a good definition for a container, and coffin doesn't belong there, cause non-chars crash the game.

Reply:
And they all have `ENT_TYPE inside`, which is the important part, they can all be used blindly exactly the same way...

Reply:
So is present actually a container, or should we remove the presents too, or just add Pot, cause they're really all the same thing anyway?

Reply:
The idea of custom types was, you can take the class name and use it as `ENT_TYPE` to get all entities as listed in `entity_hierarchy`. And also add extra ones that have similar properties etc.

present is there because this was made before I added the `Purchasable` class

So ideally this should only have: `ITEM_CRATE, ITEM_DMCRATE, ITEM_ALIVE_EMBEDDED_ON_ICE`

Reply:
Because of the exactly similar properties, and because removing present would be a breaking change anyway, I'm keeping pot.